streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -2646,6 +2646,18 @@ object ZStream extends Serializable {
   final def apply[R, E, A](pull: ZManaged[R, E, Pull[R, E, A]]): ZStream[R, E, A] =
     new ZStream[R, E, A](ZStream.Structure.Iterator(pull))
 
+  /**
+   * Creates a stream from a value received from the accessed environment.
+   */
+  final def access[R](f: R => Any): ZStream[R, Nothing, Any] =
```

Comment:
I think you want to parameterize this on `A` as well and have `f: R => A` to return a `ZStream[R, Nothing, A]`.